docs/config-file.md

```diff
@@ -31,6 +31,11 @@ Here is an example of a `glee.config.js` file for reference:
 export default async function () {
   return {
     websocket: {
+      docs: {
+        enabled: false // Enable/Disable documentation generation
+        folder: 'output' // Folder where you want the output of your docs to reside.
+        template: '@asyncapi/markdown-template' // Type of template you want to use.
+      }
```

Comment:
```suggestion
export default async function () {
  return {
        docs: {
          enabled: false // Enable/Disable documentation generation
          folder: 'output' // Folder where you want the output of your docs to reside.
          template: '@asyncapi/markdown-template' // Type of template you want to use.
      }
    websocket: {
```

It should be this right, I think docs should not go under any protocol-specific configuration, instead should be a global config. Also, I think we should by default create docs, and let user `enabled: false` it if required.

Reply:
> It should be this right, I think docs should not go under any protocol-specific configuration, instead should be a global config

Thanks for pointing that out, it was a mistake 😄

Reply:
Updated the PR @Souvikns